ingestion/src/metadata/ingestion/source/database/sample_data.py

```diff
@@ -409,83 +415,142 @@ def ingest_locations(self) -> Iterable[Location]:
-        db = Database(
-            id=uuid.uuid4(),
-            name=self.glue_database["name"],
-            description=self.glue_database["description"],
+    def ingest_glue(self):
+        db = CreateDatabaseRequest(
+            name=self.database["name"],
+            description=self.database["description"],
             service=EntityReference(
-                id=self.glue_database_service.id,
-                type=self.glue_database_service.serviceType.value,
+                id=self.database_service.id.__root__, type="databaseService"
             ),
         )
-        db_schema = DatabaseSchema(
-            id=uuid.uuid4(),
-            name=self.glue_database_schema["name"],
-            description=self.glue_database_schema["description"],
-            service=EntityReference(
-                id=self.glue_database_service.id,
-                type=self.glue_database_service.serviceType.value,
-            ),
-            database=EntityReference(id=db.id, type="database"),
+
+        yield db
+
+        database_entity = fqn.build(
+            self.metadata,
+            entity_type=Database,
+            service_name=self.database_service.name.__root__,
+            database_name=db.name.__root__,
+        )
+
+        database_object = self.metadata.get_by_name(
+            entity=Database, fqn=database_entity
+        )
+        schema = CreateDatabaseSchemaRequest(
+            name=self.database_schema["name"],
+            description=self.database_schema["description"],
+            database=EntityReference(id=database_object.id, type="database"),
+        )
+        yield schema
+
+        database_schema_entity = fqn.build(
+            self.metadata,
+            entity_type=DatabaseSchema,
+            service_name=self.database_service.name.__root__,
+            database_name=db.name.__root__,
+            schema_name=schema.name.__root__,
         )
+
+        database_schema_object = self.metadata.get_by_name(
+            entity=DatabaseSchema, fqn=database_schema_entity
+        )
+
         for table in self.glue_tables["tables"]:
-            table["id"] = uuid.uuid4()
-            parameters = table.get("Parameters")
-            table = {key: val for key, val in table.items() if key != "Parameters"}
-            table_metadata = Table(**table)
-            location_type = LocationType.Table
-            if parameters:
-                location_type = (
-                    location_type
-                    if parameters.get("table_type") != "ICEBERG"
-                    else LocationType.Iceberg
-                )
-            location_metadata = Location(
-                id=uuid.uuid4(),
+            table_request = CreateTableRequest(
                 name=table["name"],
-                path="s3://glue_bucket/dwh/schema/" + table["name"],
                 description=table["description"],
-                locationType=location_type,
+                columns=table["columns"],
```

Comment:
@MilanBariya here we are missing the information like `tableType`, `tableConstraints` which is available in [tables.json](https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/examples/sample_data/datasets/tables.json)

Reply:
I will add that.